ms2/test/src/org/labkey/test/ms2/AbstractMS2SearchEngineTest.java

```diff
@@ -129,9 +134,23 @@ public void basicMS2Check()
 
         clickAndWait(Locator.tagWithAttribute("a", "title", "Experiment run graph"));
 
-        log("Verify msPicture");
+        log("Verify graph view");
         pushLocation();
-        clickAndWait(Locator.imageMapLinkByTitle("graphmap", "Data: " + SAMPLE_BASE_NAME + ".mzXML.image..itms.png (Run Output)"));
+        Locator.linkWithSpan("Toggle Beta Graph (new!)").waitForElement(getDriver(), 4000)
+                .click();
+        LineageGraph graphComponent = new LineageGraph.LineageGraphFinder(getDriver()).waitFor();
+        List<String> actualParents = graphComponent.getDetailGroup("Data Parents")
+                .getItemNames();
+        List<String> actualChildren = graphComponent.getDetailGroup("Data Children")
+                .getItemNames();
+        assertThat(actualParents, hasItems("Bovine_mini1.fasta", "comet.xml", "CAexample_mini.mzXML"));
+        assertThat(actualChildren, hasItems("CAexample_mini.prot.xml", "CAexample_mini.pep.xml",
+                "CAexample_mini.mzXML.image..itms.png"));

```

Comment:
Are you sure that these lineages are valid for all of the subclasses of `AbstractMS2SearchEngineTest`?
If you haven't, I would suggest removing these assertions instead of updating them. We should explicitly verify the run graph somewhere rather than doing a drive-by in an arbitrary test.

Reply:
good call; we don't need to insert this into all of them